File: app/src/main/java/com/khmelenko/lab/travisclient/network/response/GithubAccessToken.java
package com.khmelenko.lab.travisclient.network.response;

import com.google.gson.annotations.SerializedName;

/**
 * DAO for Access token from Github
 *
 * @author Dmytro Khmelenko
 */
public class GithubAccessToken {

    @SerializedName("access_token")
    private String mAccessToken;

    @SerializedName("scope")
    private String mScope;

    @SerializedName("token_type")
    private String mTokenType;

    public String getAccessToken() {
        return mAccessToken;
    }

    public void setAccessToken(String accessToken) {
        mAccessToken = accessToken;
    }

    public String getScope() {
        return mScope;
    }

    public void setScope(String scope) {
        mScope = scope;
    }

    public String getTokenType() {
        return mTokenType;
    }

    public void setTokenType(String tokenType) {
        mTokenType = tokenType;
    }
}

File: app-v3/src/main/java/com/khmelenko/lab/varis/network/response/GithubAccessToken.kt
package com.khmelenko.lab.varis.network.response

import com.google.gson.annotations.SerializedName

/**
 * DAO for Access token from Github
 *
 * @author Dmytro Khmelenko
 */
data class GithubAccessToken(

        @SerializedName("access_token")
        val accessToken: String,

        @SerializedName("scope")
        val scope: String,

        @SerializedName("token_type")
        val tokenType: String
)
